src/cli/index.js

```diff
@@ -170,6 +170,7 @@ export function main({
   const run = (): Promise<void> => {
     invariant(command, 'missing command');
     return command.run(config, reporter, commander, commander.args).then(exitCode => {
+      reporter.close();
```

Comment:
Do you really need to call `close` on the reporter? It will break the `footer` call below